packages/cozy-harvest-lib/src/helpers/oauth.js

```diff
@@ -97,9 +111,7 @@ export const prepareOAuth = (client, konnector) => {
 
   const cozyUrl = client.stackClient.uri
 
-  const oAuthUrl = `${cozyUrl}/accounts/${accountType}/start?scope=${
-    oauth.scope
-  }&state=${oAuthStateKey}&nonce=${Date.now()}`
+  const oAuthUrl = getOAuthUrl(cozyUrl, accountType, oAuthStateKey, oauth, Date.now())
```

Comment:
There are too much parameters here. You should use an object (here and in the function above) for named parameters

Reply:
I had this in mind but probably was too lazy here. Will do